tests/Oryx.BuildImage.Tests/Php/PhpSampleAppsTest.cs

```diff
@@ -65,7 +74,8 @@ public void GeneratesScript_AndBuilds_WithoutComposerFile()
             var manifestFile = $"{appOutputDir}/{FilePaths.BuildManifestFileName}";
             var script = new ShellScriptBuilder()
                 .AddCommand($"rm {appDir}/composer.json")
-                .AddBuildCommand($"{appDir} -o {appOutputDir} --platform php --platform-version {PhpVersions.Php73Version}")
+                .AddCommand($"source benv php={phpVersion}")
```

Comment:
why do you need to do this? oryx build should already do this?

Reply:
you are right ... I was testing something .. this should be removed